rust/Cargo.toml

```diff
@@ -52,7 +52,8 @@ fn-ptr-conversion-inline-dispatch = ["needs-fn-ptr-conversion"]
 [dependencies]
 bnum = "0.12.0"
 ethnum = "1.5.0"
-evmc-vm-tosca = { package = "evmc-vm", git = "https://github.com/Fantom-foundation/evmc", branch = "tosca-extensions" }
+# evmc-vm-tosca = { package = "evmc-vm", git = "https://github.com/Fantom-foundation/evmc", branch = "tosca-extensions" }
+evmc-vm-tosca = { package = "evmc-vm", git = "https://github.com/LorenzSchueler/evmc", branch = "lorenz/cross-platform" }                                # TODO
```

Comment:
fixme or todo would be nice to explain the commented branch line. otherwise: dead code

Reply:
That was only because the emvc PR was not merged yet. I will update the dependencies now.